BSP/Simulator/Src/BSP_SPI.c

```diff
@@ -229,11 +248,13 @@ static void WRCommandHandler(uint8_t *buf, uint32_t len) {
         case SIM_LTC6811_WRCFGA: {
```

Comment:
Are you actually using any of the private functions?

Reply:
I commented out all the private functions and it works